.github/workflows/testAndPublish.yml

```diff
@@ -159,9 +159,19 @@ jobs:
       run: ci/scripts/setBuildVersionVars.ps1
     - name: Set scons args
       run: ci/scripts/setSconsArgs.ps1
-    - name: Prepare source code for launcher
+    - name: Install SignPath
+      if: ${{ env.feature_signing }}
```

Comment:
This if check seems inconsistent with the check that actually signs.

Reply:
I'm not sure I understand. Are you suggestion checking if the event is a push event?

Reply:
```suggestion
      if: ${{ github.event_name == 'push' && env.feature_signing }}
```

Reply:
Yes this is exactly what I meant